internal/bundle/lint_rules_policy.go

```diff
@@ -58,6 +58,7 @@ const (
 	PolicyMissingVersionRuleID       = "policy-missing-version"
 	PolicyWrongVersionRuleID         = "policy-wrong-version"
 	PolicyRequiredTagsMissingRuleID  = "policy-required-tags-missing"
+	PolicyMissingRequireRuleID       = "policy-missing-require"
```

Comment:
Do we want every user to need to think about this enough to make this a linting rule?

Reply:
We can extract this to be a separate PR and add this once we've updated all policies. I think we should encourage users to add those as it speeds up the linting as we do not need to download all providers and makes the requirements more obvious.